dev/com.ibm.websphere.appserver.features/visibility/public/jdbc-4.0/com.ibm.websphere.appserver.jdbc-4.0.feature

```diff
@@ -5,14 +5,14 @@ singleton=true
 IBM-API-Package: com.ibm.wsspi.zos.tx; type="internal"
 IBM-ShortName: jdbc-4.0
 Subsystem-Name: Java Database Connectivity 4.0
--features=com.ibm.websphere.appserver.javax.connector.internal-1.6; ibm.tolerates:=1.7, \
+-features=\
  com.ibm.websphere.appserver.jndi-1.0, \
- com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:=1.2, \
+ com.ibm.websphere.appserver.transaction-1.1; ibm.tolerates:="1.2, 2.0", \
  com.ibm.websphere.appserver.classloading-1.0, \
  com.ibm.websphere.appserver.appLifecycle-1.0, \
  com.ibm.websphere.appserver.connectionManagement-1.0, \
  com.ibm.websphere.appserver.requestProbes-1.0
--bundles=com.ibm.ws.jdbc, \
+-bundles=\
  com.ibm.ws.jdbc.4.0.feature
```

Comment:
i know i'm going to regret asking but why do we have a bundle with ".feature" in it's name? That's a bit misleading isn't it?

Reply:
com.ibm.ws.jdbc.4.0.feature is pre-existing code.  I didn't write it, nor did I add/change usage of it under this pull request, so I can only speculate about the naming of it.  My best guess would be that it is named that way because that bundle contains function that is unique to the jdbc-4.0 feature.  Similarly, the com.ibm.ws.jdbc.4.1.feature bundle contains code that is unique to the jdbc-4.1 feature.  The com.ibm.ws.jdbc.4.2.feature bundle contains code that is unique to the jdbc-4.2 feature.  The com.ibm.ws.jdbc.4.3.feature bundle contains code that is unique to the jdbc-4.3 feature.  If you have a proposal for a better name, let's do that under a separate issue/pull. It isn't related to the Jakarta work.

Reply:
It is a separate issue, it just struck me as odd when we also have bundles named `com.ibm.ws.jdbc`, `com.ibm.ws.jdbc.4.1`, `com.ibm.ws.jdbc.4.2`, and `com.ibm.ws.jdbc.4.3`

Reply:
Those bundles layer on additional function that is used by, for example, jdbc-4.1 onward.  It's no trivial task arranging the code to maintain compliant implementations of 4 different levels of the same spec at once.